ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/notification/NotificationManagerImpl.java

```diff
@@ -100,8 +100,8 @@ public NotificationManagerImpl(NotificationManagerView view,
 
     @Inject
     @PostConstruct
```

Comment:
Of course, we don't. Gin does not support `@PostConstruct` annotations. Surely it was added by a mistake. I guess there was a real need for using method injection here, though it is not actually related to this PR you can start a discussion somewhere in a mailing list for example.